execution.py

```diff
@@ -138,7 +138,8 @@ def recursive_execute(server, prompt, outputs, current_item, extra_data, execute
             server.send_sync("executing", { "node": unique_id, "prompt_id": prompt_id }, server.client_id)
 
         obj = object_storage.get((unique_id, class_type), None)
-        if obj is None:
+        should_reload = getattr(class_def, 'RELOAD_INST', False)
```

Comment:
Currently the reload of the node class is kept behind the class attribute `RELOAD_INST`. I'm not really sure if this really matters, if not, this can be removed